.timing

```diff
@@ -0,0 +1 @@
+1608152083453
```

Comment:
this file gets cleaned up as part of testing, so if it shows up in a PR, that's a sign testing went wrong.

Reply:
Sorry, I've missed your 'build' instructions on the how to contribute page. On your initial index.html question — I've tried `npm i` on my local copy, which finished with errors and `npm start`, which failed to run, presumably due to the above mentioned error. Let me investigate local install fails and get back to you with questions if unable to resolve on my own.

Reply:
I am new to online git (as oppose to local), so hopefully you can see latest commits: after fixing syntax in some graphic-element folder's files, I was able to run `npm start`(regardless of `npm i` errors), which compiled previously committed changes successfully, among else cleaning out .timing file. (In bellow commit changes resulting out of build, e.g. index.html etc are omitted, since I've read it somewhere you build via github on commits). Please provide feedback to direct me (a week ago I've sketched a half of next chapter)

Reply:
`npm i` should not have any errors, and no files should need changes in order for npm start to work: can you show my the errors you're getting during install?

Reply:
Also note that you'll want to be using the current version of Node - either the LTS version (v14) or the stable version (v15). If you're on unix/linux/macos and you need to use different versions of Node for different projects, you'll want to use [nvm](https://github.com/nvm-sh/nvm), which makes switching node versions super simple. Windows has [nvm-windows](https://github.com/coreybutler/nvm-windows) for the same purpose.

Reply:
Find attached print and log. (I did update node)
<img width="808" alt="Screen Shot 2020-12-22 at 8 29 01 PM" src="https://user-images.githubusercontent.com/66021002/102952392-adffc080-4494-11eb-9c62-6e155adadffe.png">
[2020-12-23T02_28_42_900Z-debug.log](https://github.com/Pomax/BezierInfo-2/files/5732951/2020-12-23T02_28_42_900Z-debug.log)

Reply:
Ah, that's `node-canvas`, which needs to be built in order to generate images during the build step. MacOS doesn't come with all the things that a normal unix/linux system already comes with, so there's an extra step - the official docs for that are over on https://github.com/Automattic/node-canvas#compiling but the summary is that on MacOS you'll first need to one-time run `brew install pkg-config cairo pango libpng jpeg giflib librsvg` to make sure all the graphics libs/headers etc. are installed (using the [brew](https://brew.sh/) installer)

Reply:
installed

Reply:
Nice. You should now be able to run `npm start` and that _should_ also delete this file as part of the cleanup step, after which you can do a new git commit for the build result, but if it doesn't, you'll have to manually delete this file still.

Reply:
Done.